ui/src/geometry/geometry-api.ts

```diff
@@ -300,24 +300,9 @@ export async function getTrackLayoutPlans(
     changeTime: TimeStamp,
     includeGeometryData = true,
 ): Promise<GeometryPlanLayout[]> {
-    const url = (planId: GeometryPlanId, includeGeometryData: boolean) =>
-        `${GEOMETRY_URI}/plans/layout?planIds=${planId}&includeGeometryData=${includeGeometryData}`;
-    return trackLayoutPlanCache
-        .getMany(
-            changeTime,
-            planIds,
-            (id) => `${id}-${includeGeometryData}`,
-            (fetchIds) =>
-                Promise.all(
-                    fetchIds.map((fetchId) =>
-                        getNonNull<[GeometryPlanLayout]>(url(fetchId, includeGeometryData)),
-                    ),
-                ).then((tracks) => {
-                    const trackMap = indexIntoMap(tracks.flat());
-                    return (id) => trackMap.get(id);
-                }),
-        )
-        .then((plans) => plans.filter(filterNotEmpty));
+    return Promise.all(
+        planIds.map((planId) => getTrackLayoutPlan(planId, changeTime, includeGeometryData)),
+    ).then((layouts) => layouts.filter(filterNotEmpty));
```

Comment:
Luenko tästä muutoksesta oikein että näitä ei nyt kakuteta ollenkaan, toisin kuin edellisessä versiossa? 🤔 Näitä kutsuja lähtee parhaimmillaan niin massoittain, että kakutus voisi silti olla kiva

Reply:
Ei, getTrackLayoutPlan() kakuttaa yhä.